test/std.ml

```diff
@@ -1,24 +0,0 @@
-(* check compatibility of interfaces *)
```

Comment:
You're right. It created issues because it is named the same as `Std` in extlib so the tests would accidentally pick this up, so I removed it but actually renaming is enough to work around that problem.